src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx

```diff
@@ -497,14 +472,13 @@ The PHP agent integrates with other New Relic features to give you end-to-end vi
 
     <tr>
       <td>
-        [Codestream/Code-level metrics](/docs/codestream/how-use-codestream/performance-monitoring/)
+        [Codestream (code-level metrics)](/docs/codestream/how-use-codestream/performance-monitoring/)
       </td>
 
       <td>
-        New Relic CodeStream allows developers to see how the code they're responsible for is performing in production
-        by bringing observability into the IDE and making it part of their daily routine.
-        The PHP agent reports code-level metrics (starting with release 10.4.0) on your application which enables
-        detailed insight into how your code is performing at the method level.
+        New Relic CodeStream brings observability into the IDE to show how your code is performing in production at the method level.
+
+        The PHP agent versions 7.0 - 7.4 and 8.0 - 8.2 report code-level metrics (starting with release 10.4.0).
```

Comment:
```suggestion
        PHP versions 7.0 - 7.4 and 8.0 - 8.2 report code-level metrics (starting with release 10.4.0).
```
It is versions of PHP, not the agent. Though I am unsure how necessary this list is, as it encompasses (and will continue to encompass in the future) all PHP versions supported by the agent.